Neos.EventSourcedContentRepository/Classes/Domain/ValueObject/CommandResult.php

```diff
@@ -109,20 +103,27 @@ protected static function blockProjector(DomainEvents $events, ProcessedEventsAw
                     $ids .= '   ' . $p->getIdentifier();
                 }
 
-                throw new \RuntimeException('TIMEOUT while waiting for events: ' . $ids, 1550232279);
+                throw new \RuntimeException(sprintf('TIMEOUT while waiting for event(s) %s for projector "%s"', $ids, get_class($projector)), 1550232279);
```

Comment:
Not exactly related, but with this change we see at least which projector failed